### DISCUSSION TITLE
Forge book has deal() with three parameters, but forge-std/vm.sol does not

### DISCUSSION THREAD

**Discussion by aathan** [foundry-rs/foundry#1483] [Help]
Where can I find the three parameter version of deal()?

**Comment by onbjerg** [comment]
Here: https://github.com/foundry-rs/forge-std/blob/e26ae2954bb6d883643b92760d443dd07f413647/src/Test.sol#L97

**Reply by simontianx** [reply]
There are two `deal`: one is in the `Vm` and the other is in `Test`. This raises some confusion in using `deal`. Sometimes `vm.deal` is used and sometimes a straight `deal` from Test.sol is used. Is it possible to put `deal` at one location?

**Reply by simontianx** [reply]
VM: https://github.com/foundry-rs/forge-std/blob/e26ae2954bb6d883643b92760d443dd07f413647/src/Vm.sol#L39
Test: https://github.com/foundry-rs/forge-std/blob/e26ae2954bb6d883643b92760d443dd07f413647/src/Test.sol#L97
